etc/docker/elyra_development/README.md

```diff
@@ -0,0 +1,37 @@
+<!--
+{% comment %}
+Copyright 2018-2022 Elyra Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+### Elyra Development Docker Image
+
+Builds an Elyra development workspace image. This image provides all the necessary libraries and frameworks to build and test
+Elyra changes. 
+
+#### Building the image
+
+To build a custom version of this container image:
+1. Clone this repository.
+1. [Optional] Add custom Python package requirements to `requirements.yml` in the `etc/docker/elyra_development` directory.
+1. Build the image using the `Dockerfile`:
+  - `docker build -t <DOCKER ID>/elyra-environment:dev etc/docker/elyra_development/` 
```

Comment:
I was a little confused at first what `<DOCKER ID>` was, then realized its the docker org.  Since we don't impose a restriction that the image name be `elyra-environment:dev` I think making this more free-form like:
```
docker build -t image_name:tag etc/docker/elyra_development/
```
Should probably mention that the `etc/docker/elyra_development/` portion assumes this command is invoked from the elyra root (e.g., `/path/to/elyra/root`).

It might be helpful to include an example here, like you do below, just so the all the dots can be connected.